src/i18n/es.json

```diff
@@ -1,9 +1,11 @@
 ﻿{
   "common": {
     "cancel": "Cancelar",
+    "copyright_HTML": "© {year} Doppler LLC. Todos los derechos reservados. <a target=\"_blank\" href=\"https://www.fromdoppler.com/legales/privacidad\">Política de Privacidad y Legales.</a>",
     "help": "Ayuda",
     "hide": "Ocultar",
     "message": "Mensaje",
+    "recaptcha_legal_HTML": "Este sitio es protegido por reCAPTCHA y aplican la <a href=\"https://policies.google.com/privacy?hl=es\">Política de Privacidad</a> y <a href=\"https://policies.google.com/terms?hl=es\">Condiciones del Servicio</a> de Google.",
```

Comment:
Hey @carodipietro!, I have translated it by my self, do you think that it is fine? or should I change something?

Reply:
@andresmoschini en sign up es necesario poner las políticas de google? porque en el caso de las demás herramientas que mencionamos en destinatarios no linkeamos a sus políticas de privacidad.

Reply:
En _signup_, _resend activation link_, _login_ y _forgot password_ hacemos uso de reCAPTCHA.

Según este link entiendo que en ese caso deberíamos incluir ese texto:
https://developers.google.com/recaptcha/docs/faq#id-like-to-hide-the-recaptcha-badge-what-is-allowed

Pero tal vez lo esté entendiendo mal. Si entendés que es posible omitirlo, me parece bien.

Reply:
@carodipietro por favor confirmame que hago entonces.